windows-iotcore/TOC.md

```diff
@@ -1,6 +1,6 @@
 # [Windows 10 IoT documentation](index.md)
```

Comment:
Why is windows 10 IoT doc and IoT core doc both pointing to index? We need to create a new index that talks about enterprise vs dev documentation.

Reply:
That content is in Windows 10 IoT Enterprise. This is a temporary solution at the moment and it ties into the impending homepage redesign (which will happen in 1802)!